app/build.gradle.kts

```diff
@@ -66,4 +83,17 @@ dependencies {
     androidTestImplementation("androidx.compose.ui:ui-test-junit4")
     debugImplementation("androidx.compose.ui:ui-tooling")
     debugImplementation("androidx.compose.ui:ui-test-manifest")
+
+    //Dagger - Hilt
+    implementation("com.google.dagger:hilt-android:2.44")
+    kapt("com.google.dagger:hilt-android-compiler:2.44")
```

Comment:
I recommend to use KSP (Kotlin Symbol Processing), which is a faster alternative to kapt. Recently, Dagger and Room got supported for that. 

I explain it here: https://www.youtube.com/watch?v=WVhgUGyUnhQ